### ISSUE TITLE
Make newsletter signup more prominent. 

### ISSUE THREAD

**Issue by noeljackson** [alistapart/AListApart#410]
The newsletter signup is hidden away. 

Suggestions: try it above the fold, put it in the menu, link it in the footer.

**Comment by zeldman** [comment]
Good point for possible tweak now, and certainly for upcoming redesign.
